modules/update-using-custom-machine-config-pools-inheritance.adoc

```diff
@@ -0,0 +1,206 @@
+
+
+.Prerequisites
+
+* You have created one or more custom machine config pools (MCP).
+
+.Procedure
+
+. Create a secondary MCP. 
+.. Save the following file as `machineConfigPool.yaml`.
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-perf
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {
+         key: machineconfiguration.openshift.io/role,
+         operator: In,
+         values: [worker,worker-perf]
+        }
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker-perf: "" 
+----
++
+.. Create the new machine config pool by running the following command:
++
+[source,terminal]
+----
+$ oc create -f machineConfigPool.yaml
+----
++
+.Example output
++
+[source,terminal]
+----
+machineconfigpool.machineconfiguration.openshift.io/worker-perf created
+----
+
+. Add some machines to the secondary MCP.
+This example labels the worker nodes, `worker-b` and `worker-c` to the MCP `worker-perf`:
++
+[source,terminal]
+----
+$ oc label node worker-b node-role.kubernetes.io/worker-perf=''
+$ oc label node worker-c node-role.kubernetes.io/worker-perf=''
+----
+
+. Create a new `MachineConfig` for the MCP `worker-perf`.
+.. Save the following `MachineConfig` example as `new-machineconfig.yaml`.
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: workers-perf
+  name: 06-kdump-enable-workers-perf
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    systemd:
+      units:
+      - enabled: true
+        name: kdump.service
+  kernelArguments:
+    - crashkernel=512M
+----
+
+.. Apply the `MachineConfig` by running the following command:
++
+[source,terminal]
+----
+$ oc create -f new-machineconfig.yaml
+----
+
+. Create the new canary MCP and add machines from the MCP you created above.
+This example creates the an MCP called `worker-perf-canary` MCP, and adds machines from the `worker-perf` to it. 
+
+.. Label the canary worker node `worker-a` by running the following command: 
++
+[source,terminal]
+----
+$ oc label node worker-a node-role.kubernetes.io/worker-perf-canary=''
+----
+
+.. Save the following file as `machineConfigPool-Canary.yaml`.
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-perf-canary
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {
+         key: machineconfiguration.openshift.io/role,
+         operator: In,
+         values: [worker-perf,worker-perf-canary] <1>
+        }
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker-perf-canary: ""
+----
+<1> This exmaple includes `worker-perf-canary` as an optional value. You can use a value in this way to configure members of an additional `MachineConfig`.
```

Comment:
```suggestion
<1> Optional value. The `worker-perf-canary` value is use in this example. You can use a value in this way to configure members of an additional `MachineConfig`.
```
what are valid values?

Reply:
the value here is the name of an MCP. So any MCP name is valid. What's optional here is the inclusion of an additional MCP.